x/wasm/keeper/wasmtesting/mock_engine.go

```diff
@@ -88,6 +89,20 @@ func (m *MockWasmer) Create(codeID wasmvm.WasmCode) (wasmvm.Checksum, error) {
 	return m.CreateFn(codeID)
 }
 
+func (m *MockWasmer) StoreCode(codeID wasmvm.WasmCode) (wasmvm.Checksum, error) {
```

Comment:
I was wondering if we want to use `Create` at all now in this mock. The method is required to be compatible with the interface but why not always panic with "Deprecated: use StoreCode instead"? There should not be any reason to use it in tests anymore. WDYT?

Please add a deprecated annotation to the method, too.

Reply:
Good idea!